tests/emails/beneficiary_booking_confirmation_test.py

```diff
@@ -310,3 +311,21 @@ def test_should_return_total_price_for_duo_offers():
     booking = bookings_factories.BookingFactory(quantity=2, stock__price=10)
     email_data = retrieve_data_for_beneficiary_booking_confirmation_email(booking)
     assert email_data["Vars"]["offer_price"] == "20.00 €"
+
+
+@freeze_time("2021-07-01 10:00:00")
+@pytest.mark.usefixtures("db_session")
+def test_digital_offer_without_departement_code_information():
+    """
+    Test that a user without any postal code information can book a digital
+    offer. The booking date information should use the default timezone:
+    metropolitan France.
+    """
+    offer = offers_factories.DigitalOfferFactory()
+    stock = offers_factories.StockFactory(offer=offer, price=10)
```

Comment:
(détail)

pas besoin de spécifier le prix, ça n'affecte pas le test

```suggestion
    stock = offers_factories.StockFactory(offer=offer)
```